pkg/kubelet/prober/worker.go

```diff
@@ -152,9 +152,9 @@ func (w *worker) doProbe() (keepGoing bool) {
 	}
 
 	c, ok := api.GetContainerStatus(status.ContainerStatuses, w.container.Name)
-	if !ok {
+	if !ok || len(c.ContainerID) == 0 {
 		// Either the container has not been created yet, or it was deleted.
```

Comment:
@timstclair Is there ever a case where a started-then-deleted container would hit this? If so, then saying "Unstarted container probed" wouldn't be 100% accurate...